README.rst

```diff
@@ -1,3 +1,19 @@
+Eventlet
+========
+
+.. image:: https://img.shields.io/pypi/v/eventlet
+    :target: https://pypi.org/project/eventlet/
+
+.. image:: https://img.shields.io/github/actions/workflow/status/eventlet/eventlet/test.yaml?branch=master
+    :target: https://github.com/eventlet/eventlet/actions?query=workflow%3Atest+branch%3Amaster
+
+.. image:: https://codecov.io/gh/eventlet/eventlet/branch/master/graph/badge.svg
+    :target: https://codecov.io/gh/eventlet/eventlet
+
+.. warning::
```

Comment:
Looks like rst notes and warnings [1] are not recognized by github's rendering engine. Do you think we should simply bold the following sentence?

[1] https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
How about a) make "Warning" a chapter header? b) move this even in front of the "Eventlet" chapter? After all, this likely is the most important information on this page for anyone reading this doc for the first time.

Reply:
Good idea. I'll update the patch to match your proposal. Will do that after my PTO.